src/dxgi/dxgi_adapter.cpp

```diff
@@ -283,6 +283,15 @@ namespace dxvk {
       else
         sharedMemory += heap.size;
     }
+
+    // Some games think we are on Intel given a lack of
+    // NVAPI or AGS/atiadlxx support.
+    // Report our device memory as shared memory,
+    // and some small amount for the carveout.
+    if (options->emulateUMA && !m_adapter->isUnifiedMemoryArchitecture()) {
+      deviceMemory = 128 * (1 << 20);
+      sharedMemory = deviceMemory;
```

Comment:
surely this should be the other way around? We're now reporting 128MB for *both* shared and device memory.

Reply:
Yes this should :frog: